### PR TITLE
Task/OB-30

### PR DESCRIPTION
**Pull request by csabbee**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -136,60 +138,47 @@ gulp.task('sass', () => {
...
+// When running 'gulp' on the terminal this task will fire.
+// It will start watching for changes in every .js file.
+// If there's a change, the task 'scripts' defined above will fire.
+gulp.task('default', sequence(
+    ['clean-js', 'clean-css'],
+    ['copy:index.html', 'copy:fonts', 'sass', 'scripts:vendor', 'scripts:app', 'backend'],
+    'index',
+    'connect',
+    ['watch', 'watch:backend']
+));
```
Comment (ppseprus) [inline gulpfile.js]: For the sake of consistency, please rename "clean-js" to "clean:js" and "clean-css" to "clean:css".
Also, we have a task named _index_, so why not name the other "copy:index"?
#niceToHave

**Review by ppseprus — APPROVED** [review]

File: gulpfile.js (hunk 1 above)
Comment (csabbee) [reply gulpfile.js]: This isn't a nice to have, but rather solid points

Comment (ppseprus) [reply gulpfile.js]: It's my comment! I decide that! :D

**Review by ppseprus — CHANGES_REQUESTED** [review]
Still not solving the issue... : /

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node node_modules/gulp/bin/gulp.js --color"
+    "start": "set NODE_ENV=dev && node node_modules/gulp/bin/gulp.js --color",
+    "deploy": "set NODE_ENV=prod && node node_modules/gulp/bin/gulp.js deploy --color"
```
Comment (ppseprus) [inline package.json:9]: You do not have to set/export. The proper way of using NODE_ENV is `NODE_ENV={env} node app.js`

Comment (csabbee) [reply package.json:9]: In windows that is not working

Comment (ppseprus) [reply package.json:9]: Strange... That is the only way I've found it on SO.

**Comment by ppseprus** [comment]
Nicely done! Now, it works perfectly!
Theres one last thing we might want to consider: when the gulpfile is run without the NODE_ENV... We should throw a console log and exit out. Also, if it does not match any of the defined types.
What do you think?

**Review by ppseprus — APPROVED** [review]

**Comment by ppseprus** [comment]
+1

**Pull request merged**